AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/RequestController.cs

```diff
@@ -98,21 +102,28 @@ public RequestController(IMediator mediator)
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> Edit(EditRequestViewModel model)
         {
-            var campaignEvent = await _mediator.SendAsync(new EventSummaryQuery { EventId = model.EventId });
+            if (!ModelState.IsValid)
+            {
+                return View("Edit", model);
+            }
 
-            if (campaignEvent == null)
+            var @event = await _mediator.SendAsync(new EventSummaryQuery { EventId = model.EventId });
+            if (@event == null)
             {
                 return BadRequest("Invalid event id");
             }
 
-            if (!User.IsOrganizationAdmin(campaignEvent.OrganizationId))
+            if (!User.IsOrganizationAdmin(@event.OrganizationId))
             {
                 return Unauthorized();
             }
 
-            if (!ModelState.IsValid)
+            var validatePhoneNumberResult = await _mediator.SendAsync(new ValidatePhoneNumberRequest { PhoneNumber = model.Phone, ValidateType = true });
+            if (!validatePhoneNumberResult.IsValid)
             {
-                return View("Edit", model);
+                var phoneAttribute = model.GetType().GetProperties().Single(p => p.Name == nameof(model.Phone)).GetCustomAttributes(false).Cast<Attribute>().Single(a => a is PhoneAttribute) as PhoneAttribute;
```

Comment:
I was thinking for now just something like

```
if (!validatePhoneNumberResult.IsValid)
{
	ModelState.AddModelError(nameof(model.Phone), "Unable to validate phone number");
	return View("Edit", model);
}
```

The prior validation the Phone attribute gives use triggers prior to getting to the Twiliio code so at this point we can just add whatever error message we want.